pkgs/development/perl-modules/Bio-BigFile/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, perl, buildPerlModule, fetchpatch, fetchurl, fetchFromGitHub, kent,
+  BioPerl, IOString, libpng, libmysqlclient, openssl }:
+# This version of bio::DB::bigfile is suited for ENSEMBL VEP
+# See https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile
+
+# Only kent 335 works with Bio-BigFile. The kent package on nix must be slightly changed
+# 1. building utils fails (see patche below)
+# 2. we need headers (PR submitted for kent package)
+let
+  kent335 = kent.overrideAttrs (old: rec {
```

Comment:
Is there a link to an upstream issue regarding Bio-BigFile not working with the latest `kent`? If so, put such links in comments.